File: GRLTest/MultirootmodeL/MultirootmodeL.java
public class Model{
	public double MultirootmodeL(double J,double G){
		double expr = 0.25*Math.max(0, Math.min(100.0, Math.min(-0.25*G, 0.25*J))) + 0.75*Math.max(0, Math.min(100.0, Math.min(0.25*G, 0.75*J)));
		return expr;
	}
}

File: GRLTest/MultirootmodeL/MultirootmodeL.m
function expr=MultirootmodeL( J,G)
	expr = 0.25*max(0, min(100.0, min(-0.25*G, 0.25*J))) + 0.75*max(0, min(100.0, min(0.25*G, 0.75*J)))
end
